web/src/emoji_picker.ts

```diff
@@ -757,7 +850,10 @@ function register_click_handlers() {
         e.stopPropagation();
 
         const compose_click_target = compose_ui.get_compose_click_target(e);
-        if ($(compose_click_target).parents(".message_edit_form").length === 1) {
+        if (
+            $(compose_click_target).parents(".message_edit_form").length === 1 &&
+            compose_click_target instanceof HTMLElement
+        ) {
```

Comment:
This is wrong -- `get_compose_click_target` is probably mistyped and should return `HTMLElement`; fix this in a prep commit.

Reply:
`get_compose_click_target` returns `compose_control_buttons_popover.reference` which has its type as `ReferenceElement extends Element` from tippy.js. 
So, to change the return type, I would need to include in the condition 
`compose_control_buttons_popover.reference instanceof HTMLElement`

Is this the right way to do this?

Reply:
I'm not sure, maybe post in #frontend for advice on this sort of detail.